module_utils/ansible_nailgun_cement.py

```diff
@@ -426,6 +432,10 @@ def find_template_input(module, name, template, failsafe=True):
     return handle_find_response(module, response, message="No template input found for <%s> in job template" % name, failsafe=failsafe)
 
 
+def find_operating_systems(module, title):
```

Comment:
This should really be `titles` (https://en.wiktionary.org/wiki/title).
Also `find_operating_systems_by_title` could be instructive, because all other 'find_' routines look for names.